libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Spacings } from '@monorepo/expo/shared/static';
+import { Accordion, Textarea } from '@monorepo/expo/shared/ui-components';
+import { RefObject } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { ScrollView, View } from 'react-native';
+import {
+  CreateClientProfileInput,
+  UpdateClientProfileInput,
+} from '../../../apollo';
+
+interface IImportantNotesProps {
+  expanded: undefined | string | null;
+  setExpanded: (expanded: undefined | string | null) => void;
+  scrollRef: RefObject<ScrollView>;
+}
+
+export default function ImportantNotes(props: IImportantNotesProps) {
+  const { scrollRef, expanded, setExpanded } = props;
+  const { control } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
+
+  const isImportantNotes = expanded === 'Important Notes';
```

Comment:
Seems like this `const` actually tracks whether Important Notes is currently expanded. Perhaps rename it to `isExpanded`?

Reply:
everywhere is "expanded" we can do this in code cleaning as well. Not to touch all files for this PR

Reply:
created follow up ticket https://betterangels.atlassian.net/browse/DEV-861